cpp/src/parquet/arrow/writer.h

```diff
@@ -211,8 +211,9 @@ inline void ArrowTimestampToImpalaTimestamp(const int64_t time, Int96* impala_ti
   (*impala_timestamp).value[2] = (uint32_t)julian_days;
 
   int64_t last_day_units = time % UnitPerDay;
-  int64_t* impala_last_day_nanos = reinterpret_cast<int64_t*>(impala_timestamp);
-  *impala_last_day_nanos = last_day_units * NanosecondsPerUnit;
+  auto last_day_nanos = last_day_units * NanosecondsPerUnit;
+  // Strage might be unaligned, so use mempcy instead of reinterpret_cast
```

Comment:
All even indexed `Int96` in a vector will be unaligned (according to int64_t alignment).

Reply:
Good point I'll open up a follow-up PR, we must not hav good test data here or UBSan isn't foolproof, or somehow I didn't run this test properly

Reply:
Wait were you just commenting on my comment?

Reply:
Yes, I was commenting on your comment on the "strange" part :)

Reply:
Did you notice the typo? ("Strage")